app/views/layouts/_flash_message.html.erb

```diff
@@ -1,6 +1,6 @@
 <%# locals: (message:) %>
 
-<div class="flex flex-col items-center justify-center fixed bottom-[64px] w-full z-30 px-5">
+<div class="flex flex-col items-center justify-center fixed bottom-[64px] pb-safe-area-1 w-full z-30 px-5">
```

Comment:
I think this is a good opportunity to turn `bottom-[64px]` into a class that also takes the safe area into account. Something like `bottom-safe-area-navbar`. It also won't be necessary to add padding.